injective/asset.json

```
@@ -417,34 +465,39 @@
     "name": "Wrapped Ethereum",
     "symbol": "WETH",
     "decimals": "18",
-    "type": "erc20"
+    "type": "peggy",
+    "icon": "https://explorer.injective.network/vendor/@injectivelabs/token-metadata/ethereum.svg"
   },
   {
     "id": "factory/inj14ejqjyq8um4p3xfqj74yld5waqljf88f9eneuk/inj1dxv423h8ygzgxmxnvrf33ws3k94aedfdevxd8h",
     "name": "Wrapped Matic (Wormhole)",
     "symbol": "WMATIC",
     "decimals": "8",
-    "type": "tokenfactory"
+    "type": "tokenfactory",
+    "icon": "https://explorer.injective.network/vendor/@injectivelabs/token-metadata/polygon.png"
   },
   {
     "id": "factory/inj14ejqjyq8um4p3xfqj74yld5waqljf88f9eneuk/inj1sthrn5ep8ls5vzz8f9gp89khhmedahhdkqa8z3",
     "name": "Wrapped SOL",
     "symbol": "SOL",
     "decimals": "8",
-    "type": "tokenfactory"
+    "type": "tokenfactory",
+    "icon": "https://explorer.injective.network/vendor/@injectivelabs/token-metadata/solana.svg"
   },
   {
     "id": "peggy0x080B12E80C9b45e97C23b6ad10a16B3e2a123949",
     "name": "XBX",
     "symbol": "XBX",
     "decimals": "18",
-    "type": "erc20"
+    "type": "peggy",
+    "icon": "https://explorer.injective.network/vendor/@injectivelabs/token-metadata/xbx.png"
   },
   {
     "id": "helix/ec5d399846a9209f3fe5881d70aae9268c94339ff9817e8d18ff19fa05eea1c8",
     "name": "XRP",
     "symbol": "XRP",
     "decimals": "6",
-    "type": "oracle"
+    "type": "oracle",
+    "icon": "https://explorer.injective.network/vendor/@injectivelabs/token-metadata/solana.svg"
```

```suggestion
    "icon": "https://cryptologos.cc/logos/xrp-xrp-logo.svg"
```

should be the xrp icon, not sure if there's an `injective.network` endpoint version for this